herwigpp.spec

```diff
@@ -1,35 +1,89 @@
-### RPM external herwigpp 2.7.1
-Source: http://service-spi.web.cern.ch/service-spi/external/MCGenerators/distribution/herwig++/herwig++-%{realversion}-src.tgz
-Requires: boost thepeg gsl hepmc
+### RPM external herwigpp 7.1.2
```

Comment:
I understand this to be herwig rather than herwig++ - why not name the spec file accordingly?